otx/cli/tools/export.py

```diff
@@ -40,6 +40,11 @@ def get_args():
         "--save-model-to",
         help="Location where exported model will be stored.",
     )
+    parser.add_argument(
+        "--dump_features",
+        action="store_true",
+        help="Whether to return feature vector and saliency map for explanation purposes.",
+    )
```

Comment:
Personally, like Galina did, I would also default it to False (to make IR infer fast by default). But not sure about that.

What do you think @goodsong81?

Reply:
I agree w/ default=False.

Reply:
I also agree, for classification, IR inference time was quiet long in proportion to dataset size.